doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,317 @@
+
+    * :ref:`central_bas`
+    * :ref:`bluetooth_central_hids`
+    * :ref:`peripheral_lbs`
+    * :ref:`shell_bt_nus`
+
+  * Other:
+
+    * :ref:`ppi_trace_sample`
+
+* Added or updated documentation for the following libraries:
+
+  * nRF9160:
+
+    * :ref:`lib_spm`
+    * :ref:`at_cmd_readme`
+    * :ref:`lib_download_client`
+    * :ref:`lib_aws_fota`
+    * :ref:`lib_fota_download`
+    * :ref:`lib_secure_services`
+
+  * Bluetooth Low Energy:
+
+    * :ref:`bas_c_readme`
+
+  * Other:
+
+    * :ref:`ppi_trace`
+    * :ref:`ble_console_readme`
+
+* Added or updated the following documentation:
+
+  * :ref:`ug_bt_ll_nrfxlib`
+  * :ref:`ug_multi_image`
+  * :ref:`partition_manager`
+  * :ref:`nrf_desktop`
+  * :ref:`shell_bt_nus_readme`
+
+* API documentation of all libraries now also mentions the location of header files and source files.
+
+Known issues
+************
+
+nRF9160
+=======
+
+* The :ref:`asset_tracker` sample does not wait for connection to nRF Cloud before trying to send data.
+  This causes the sample to crash if the user toggles one of the switches before the board is connected to the cloud.
+* The :ref:`asset_tracker` sample might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
+* If a debugger (for example, J-Link) is connected via SWD to the nRF9160, the modem firmware will reset.
+  Therefore, the LTE modem cannot be operational during debug sessions.
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+* nRF91 fails to receive large packets (over 4000 bytes) over NB-IoT. LTE-M is not affected.
+* nrf_connect fails if called immediately after initialization of the device. A delay of 1000 ms is required for this to work as intended.
+
+Crypto
+======
+
+* The :ref:`nrfxlib:nrf_security` glue layer is broken because symbol renaming is not handled correctly.
+  Therefore, the behavior is undefined when selecting multiple back-ends for the same algorithm (for example, AES).
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* :ref:`peripheral_lbs` does not report the Button 1 state correctly.
+* The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+* On some operating systems, the nrf_desktop application is unable to reconnect to a host.
+* central_uart: A too long 212-byte string cannot be handled when entered to the console to send to peripheral_uart.
+* central_hids: After flashing a HEX file to the nrf52_pca10040 board, UART connectivity is lost when using the BLE Controller. The board must be reset to get UART output.
```

Comment:
Please add one more item here:

* peripheral_hids_mouse: The sample may crash on the nRF51 device when HIDs notifications are sent frequently with button presses. In this case, the GPIO ISR introduces latency that violates real-time requirements of the Radio ISR. The nRF51 is more sensitive to this issue than the nRF52 (faster Core).

Reply:
done

Reply:
BLE examples located in fw_nrf_connect_nrf might not work as expected on PCA 10028 / nRF51

Reply:
My understanding was that this problem applies to ALL Nordic BLE examples located in our NCS repo

Reply:
Potentially all samples that use GPIO, because a user can spam application with button presses and delay the Radio ISR.

UART, throughput samples are exceptions, they don't use buttons.

Reply:
I had a short call with @shantha-14 @b-gent and we discussed how to change this note.

@b-gent should post our suggestion soon.

Reply:
Changed to:
On nRF51 devices, BLE samples that use GPIO might crash when HIDs notifications are sent frequently using button presses. In such case, the GPIO ISR introduces latency that violates real-time requirements of the Radio ISR. nRF51 is more sensitive to this issue than nRF52 (faster core).

@kapi-no  @shankar-vv  please verify

Reply:
Maybe, we can make the first sentence more general if we refer to all BLE samples:

might crash when HIDs notifications are sent frequently using button presses.
->
might crash when buttons are pressed frequently.

Reply:
yes, this makes much more sense :)